app/settings/surveys/attribute-editor.directive.js

```diff
@@ -77,6 +77,169 @@ function (
                 return $scope.editAttribute.type === 'media' && $scope.editAttribute.input === 'upload';
             };
 
+            $scope.canDisplayDefaultValue = function () {
+                return $scope.editAttribute.input !== 'upload' && $scope.editAttribute.type !== 'title' && $scope.editAttribute.type !== 'description' && $scope.editAttribute.input !== 'tags' && $scope.editAttribute.input !== 'location' && $scope.editAttribute.input !== 'relation';
+            };
+
+            $scope.displayVideoLabel = function () {
```

Comment:
@Ifycode There is a fix checking the what field-type is present in this directive (it was added after you started working with this). You will be able to get the type of field from there.